src/Components/Calculator.js

```diff
@@ -1,5 +1,6 @@
 import React, { useState } from 'react';
 import calculate from '../logic/calculate';
+import styles from '../styles/Calculator.module.css';
 
 const Calculator = () => {
   const [state, setState] = useState({
```

Comment:
While trying to run your app it was complied with the following errors:

![Screenshot 2023-03-03 at 12 45 01 PM](https://user-images.githubusercontent.com/79506105/222701237-51429612-6b82-4311-9727-35e62656613e.png)

This error is caused due to the importation path you used for you your styles. your styling files are saved in small letters but when you were importing them to your Component files you started with a capital letter

use this example to import your styling in all your component files.

import styles from '../styles/calculator.module.css';
import styles from '../styles/header.module.css';
import styles from '../styles/home.module.css';

Please lets pay attention to detail follow the above example and your app will work just fine.